security_monkey/watchers/s3.py

```diff
@@ -58,6 +62,7 @@ def slurp_items(**kwargs):
 
             for bucket_name in bucket_names:
                 bucket = self.process_bucket(bucket_name, name=bucket_name, **kwargs)
+
                 if bucket:
```

Comment:
Just now running into issues here.  Appears that bucket, on L66, is always `None`.  This looks to be caused by the removal of the `return` object in the `process_bucket` def above.